### PR TITLE
en: Update descriptions about 迁移 & 同步 to make it clearer (#312)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #312 to release-2.0

---

### What is changed, added or deleted? (Required)

To make the description about 迁移&同步 clearer:
Chinese：（全量）迁移、（增量）复制、（全量+增量模式）迁移
English：migrate, replicate, migrate

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/306
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [ ] Have version specific changes 
- [x] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/feature-shard-merge.md [hunk 1]
```diff
@@ -1,35 +1,41 @@
 ---
+<<<<<<< HEAD
 title: Merge and Replicate Data from Sharded Tables
 summary: Learn how DM merges and replicates data from sharded tables.
+=======
```
Comment (yikeke) [inline en/feature-shard-merge.md:5]: Please resolve all conflicts. @ireneontheway

File: en/feature-shard-merge.md [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
+title: Merge and Migrate Data from Sharded Tables
+summary: Learn how DM merges and migrates data from sharded tables.
+aliases: ['/docs/tidb-data-migration/dev/feature-shard-merge/']
```
Comment (ireneontheway) [inline en/feature-shard-merge.md:8]:
```suggestion
aliases: ['/docs/tidb-data-migration/v2.0/feature-shard-merge/']
```

File: en/feature-shard-merge.md [hunk 3]
```diff
@@ -1,35 +1,41 @@
...
+>>>>>>> e32acdc... en: Update descriptions about 迁移 & 同步 to make it clearer (#312)
```
Comment (ireneontheway) [inline en/feature-shard-merge.md:9]:
```suggestion
```

File: en/feature-shard-merge.md [hunk 4]
```diff
@@ -1,35 +1,41 @@
 ---
+<<<<<<< HEAD
```
Comment (ireneontheway) [inline en/feature-shard-merge.md:2]:
```suggestion
```

File: en/feature-shard-merge.md [hunk 5]
```diff
@@ -1,35 +1,41 @@
...
 title: Merge and Replicate Data from Sharded Tables
```
Comment (ireneontheway) [inline en/feature-shard-merge.md:3]:
```suggestion
```

File: en/feature-shard-merge.md [hunk 6]
```diff
@@ -1,35 +1,41 @@
...
 summary: Learn how DM merges and replicates data from sharded tables.
```
Comment (ireneontheway) [inline en/feature-shard-merge.md:4]:
```suggestion
```

File: en/feature-shard-merge.md (hunk 1 above)
Comment (ireneontheway) [inline en/feature-shard-merge.md:5]:
```suggestion
```

File: en/migrate-data-using-dm.md [hunk 7]
```diff
@@ -1,11 +1,17 @@
 ---
+<<<<<<< HEAD:en/replicate-data-using-dm.md
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:2]:
```suggestion
```

File: en/migrate-data-using-dm.md [hunk 8]
```diff
@@ -1,11 +1,17 @@
...
 title: Replicate Data Using Data Migration
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:3]:
```suggestion
```

File: en/migrate-data-using-dm.md [hunk 9]
```diff
@@ -1,11 +1,17 @@
...
 summary: Use the Data Migration tool to replicate the full data and the incremental data.
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:4]:
```suggestion
```

File: en/migrate-data-using-dm.md [hunk 10]
```diff
@@ -1,11 +1,17 @@
...
+=======
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:5]:
```suggestion
```

File: en/migrate-data-using-dm.md [hunk 11]
```diff
@@ -1,11 +1,17 @@
...
+title: Migrate Data Using Data Migration
+summary: Use the Data Migration tool to migrate the full data and the incremental data.
+aliases: ['/docs/tidb-data-migration/dev/replicate-data-using-dm/','/tidb-data-migration/dev/replicate-data-using-dm']
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:8]:
```suggestion
aliases: ['/docs/tidb-data-migration/v2.0/replicate-data-using-dm/','/tidb-data-migration/v2.0/replicate-data-using-dm']
```

File: en/migrate-data-using-dm.md [hunk 12]
```diff
@@ -1,11 +1,17 @@
...
+>>>>>>> e32acdc... en: Update descriptions about 迁移 & 同步 to make it clearer (#312):en/migrate-data-using-dm.md
```
Comment (ireneontheway) [inline en/migrate-data-using-dm.md:9]:
```suggestion
```

File: en/monitor-a-dm-cluster.md [hunk 13]
```diff
@@ -1,11 +1,16 @@
 ---
 title: Data Migration Monitoring Metrics
+<<<<<<< HEAD
```
Comment (ireneontheway) [inline en/monitor-a-dm-cluster.md:3]:
```suggestion
```

File: en/monitor-a-dm-cluster.md [hunk 14]
```diff
@@ -1,11 +1,16 @@
...
 summary: Learn about the monitoring metrics when you use Data Migration to replicate data.
```
Comment (ireneontheway) [inline en/monitor-a-dm-cluster.md:4]:
```suggestion
```

File: en/monitor-a-dm-cluster.md [hunk 15]
```diff
@@ -1,11 +1,16 @@
...
+=======
```
Comment (ireneontheway) [inline en/monitor-a-dm-cluster.md:5]:
```suggestion
```

File: en/monitor-a-dm-cluster.md [hunk 16]
```diff
@@ -1,11 +1,16 @@
...
+summary: Learn about the monitoring metrics when you use Data Migration to migrate data.
+aliases: ['/docs/tidb-data-migration/dev/monitor-a-dm-cluster/']
+>>>>>>> e32acdc... en: Update descriptions about 迁移 & 同步 to make it clearer (#312)
```
Comment (ireneontheway) [inline en/monitor-a-dm-cluster.md:8]:
```suggestion
```

File: en/monitor-a-dm-cluster.md [hunk 17]
```diff
@@ -1,11 +1,16 @@
 ---
 title: Data Migration Monitoring Metrics
+<<<<<<< HEAD
 summary: Learn about the monitoring metrics when you use Data Migration to replicate data.
+=======
+summary: Learn about the monitoring metrics when you use Data Migration to migrate data.
+aliases: ['/docs/tidb-data-migration/dev/monitor-a-dm-cluster/']
```
Comment (ireneontheway) [inline en/monitor-a-dm-cluster.md:7]:
```suggestion
aliases: ['/docs/tidb-data-migration/v2.0/monitor-a-dm-cluster/']
```

File: en/usage-scenario-simple-migration.md [hunk 18]
```diff
@@ -1,11 +1,16 @@
 ---
 title: Data Migration Simple Usage Scenario
+<<<<<<< HEAD:en/usage-scenario-simple-replication.md
```
Comment (ireneontheway) [inline en/usage-scenario-simple-migration.md:3]:
```suggestion
```

File: en/usage-scenario-simple-migration.md [hunk 19]
```diff
@@ -1,11 +1,16 @@
...
 summary: Learn how to use Data Migration to replicate data in a simple scenario.
```
Comment (ireneontheway) [inline en/usage-scenario-simple-migration.md:4]:
```suggestion
```

File: en/usage-scenario-simple-migration.md [hunk 20]
```diff
@@ -1,11 +1,16 @@
...
+=======
```
Comment (ireneontheway) [inline en/usage-scenario-simple-migration.md:5]:
```suggestion
```

File: en/usage-scenario-simple-migration.md [hunk 21]
```diff
@@ -1,11 +1,16 @@
...
+summary: Learn how to use Data Migration to migrate data in a simple scenario.
+aliases: ['/docs/tidb-data-migration/dev/usage-scenario-simple-replication/','/tidb-data-migration/dev/usage-scenario-simple-replication']
```
Comment (ireneontheway) [inline en/usage-scenario-simple-migration.md:7]:
```suggestion
aliases: ['/docs/tidb-data-migration/v2.0/usage-scenario-simple-replication/','/tidb-data-migration/v2.0/usage-scenario-simple-replication']
```

File: en/usage-scenario-simple-migration.md [hunk 22]
```diff
@@ -1,11 +1,16 @@
...
+>>>>>>> e32acdc... en: Update descriptions about 迁移 & 同步 to make it clearer (#312):en/usage-scenario-simple-migration.md
```
Comment (ireneontheway) [inline en/usage-scenario-simple-migration.md:8]:
```suggestion
```

File: en/relay-log.md [hunk 23]
```diff
@@ -1,6 +1,11 @@
 ---
 title: Data Migration Relay Log
+<<<<<<< HEAD
```
Comment (ireneontheway) [inline en/relay-log.md:3]:
```suggestion
```

File: en/relay-log.md [hunk 24]
```diff
@@ -1,6 +1,11 @@
...
 summary: Learn the directory structure, initial replication rules and data purge of DM relay logs.
```
Comment (ireneontheway) [inline en/relay-log.md:4]:
```suggestion
```

File: en/relay-log.md [hunk 25]
```diff
@@ -1,6 +1,11 @@
...
+=======
```
Comment (ireneontheway) [inline en/relay-log.md:5]:
```suggestion
```

File: en/relay-log.md [hunk 26]
```diff
@@ -1,6 +1,11 @@
...
+summary: Learn the directory structure, initial migration rules and data purge of DM relay logs.
+aliases: ['/docs/tidb-data-migration/dev/relay-log/']
```
Comment (ireneontheway) [inline en/relay-log.md:7]:
```suggestion
aliases: ['/docs/tidb-data-migration/v2.0/relay-log/']
```

File: en/relay-log.md [hunk 27]
```diff
@@ -1,6 +1,11 @@
...
+>>>>>>> e32acdc... en: Update descriptions about 迁移 & 同步 to make it clearer (#312)
```
Comment (ireneontheway) [inline en/relay-log.md:8]:
```suggestion
```

File: en/feature-shard-merge.md (hunk 1 above)
Comment (ireneontheway) [reply en/feature-shard-merge.md]: Done in https://github.com/pingcap/docs-dm/pull/352/commits/715a1b4aedb19626ab504b1e0c87a81ceb72e67a

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Comment by ireneontheway** [comment]
@yikeke PTAL

File: en/feature-shard-merge.md [hunk 28]
```diff
@@ -1,35 +1,36 @@
 ---
-title: Merge and Replicate Data from Sharded Tables
-summary: Learn how DM merges and replicates data from sharded tables.
+title: Merge and Migrate Data from Sharded Tables
+summary: Learn how DM merges and migrates data from sharded tables.
+aliases: ['/docs/tidb-data-migration/v2.0/feature-shard-merge/']
```
Comment (yikeke) [inline en/feature-shard-merge.md:4]:
```suggestion
```
no need to add this alias

File: en/migrate-data-using-dm.md [hunk 29]
```diff
@@ -1,11 +1,12 @@
 ---
-title: Replicate Data Using Data Migration
-summary: Use the Data Migration tool to replicate the full data and the incremental data.
+title: Migrate Data Using Data Migration
+summary: Use the Data Migration tool to migrate the full data and the incremental data.
+aliases: ['/docs/tidb-data-migration/v2.0/replicate-data-using-dm/','/tidb-data-migration/v2.0/replicate-data-using-dm']
```
Comment (yikeke) [inline en/migrate-data-using-dm.md:4]:
```suggestion
aliases: ['/tidb-data-migration/v2.0/replicate-data-using-dm']
```

File: en/monitor-a-dm-cluster.md [hunk 30]
```diff
@@ -1,11 +1,12 @@
 ---
 title: Data Migration Monitoring Metrics
-summary: Learn about the monitoring metrics when you use Data Migration to replicate data.
+summary: Learn about the monitoring metrics when you use Data Migration to migrate data.
+aliases: ['/docs/tidb-data-migration/v2.0/monitor-a-dm-cluster/']
```
Comment (yikeke) [inline en/monitor-a-dm-cluster.md:4]:
```suggestion
```

File: en/relay-log.md [hunk 31]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Data Migration Relay Log
-summary: Learn the directory structure, initial replication rules and data purge of DM relay logs.
+summary: Learn the directory structure, initial migration rules and data purge of DM relay logs.
+aliases: ['/docs/tidb-data-migration/v2.0/relay-log/']
```
Comment (yikeke) [inline en/relay-log.md:4]:
```suggestion
```

File: en/usage-scenario-simple-migration.md [hunk 32]
```diff
@@ -1,11 +1,12 @@
 ---
 title: Data Migration Simple Usage Scenario
-summary: Learn how to use Data Migration to replicate data in a simple scenario.
+summary: Learn how to use Data Migration to migrate data in a simple scenario.
+aliases: ['/docs/tidb-data-migration/v2.0/usage-scenario-simple-replication/','/tidb-data-migration/v2.0/usage-scenario-simple-replication']
```
Comment (yikeke) [inline en/usage-scenario-simple-migration.md:4]:
```suggestion
aliases: ['/tidb-data-migration/v2.0/usage-scenario-simple-replication']
```

**Review by yikeke — COMMENTED** [review]
@ireneontheway You added some unnecessary aliases:

**Review by yikeke — APPROVED** [review]

**Pull request merged**